Stripe/PublicHeaders/STPAPIClient.h

```diff
@@ -55,7 +55,7 @@ typedef void (^STPCompletionBlock)(STPToken * __nullable token, NSError * __null
 @property (nonatomic, copy, nullable) NSString *publishableKey;
 
 /**
- *  The operation queue on which to run the url connection and delegate methods. Cannot be nil. @see NSURLConnection
+ *  The operation queue on which to run completion blocks passed to the api client. Cannot be nil.
```

Comment:
nit: can probably remove "Cannot be nil" now that there's a `nonnull` annotation